packages/app/src/components/AffineEditor/AffineEditor.tsx

```diff
@@ -320,8 +321,19 @@ export function AffineEditor(props: {
 }) {
   const [div, setDiv] = createSignal<HTMLDivElement>()
```

Comment:
Perhaps editorCardDiv as a name? Or holder, or whats the use of it?

Reply:
more like the whole `div` for the element, its used as event target for `WheelZoomCamera2D`